docs/UserGuide.adoc

```diff
@@ -47,214 +47,354 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-* The index refers to the index number shown in the displayed person list.
-* The index *must be a positive integer* `1, 2, 3, ...`
+* Edits the delivery man at the specified `INDEX`. The index refers to the index number shown by `/deilveryman`. The index *must be a positive integer* 1, 2, 3, ...
+* At least one of the optional fields must be provided.
+* Existing values will be updated to the input values.
+* When a parameter is specified, e.g. `n/`, `p/` empty fields are not allowed. A value must be specified.
 ****
 
 Examples:
 
-* `list` +
-`select 2` +
-Selects the 2nd person in the address book.
-* `find Betsy` +
-`select 1` +
-Selects the 1st person in the results of the `find` command.
+* `/deliveryman` +
+`/deliveryman edit 1 n/John Doe p/12345678` +
+Edits the name and phone number of the 1st delivery man to be `John Doe` and `12345678` respectively.
 
-=== Listing entered commands : `history`
+==== Finding delivery man by name : `/deliveryman find`
 
-Lists all the commands that you have entered in reverse chronological order. +
-Format: `history`
+Find delivery men whose name contain in the given parameter. +
+Format: `/order find NAME [MORE NAMES] +
 
-[NOTE]
-====
-Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
-====
+****
+* The search is case insensitive. e.g `tom` will match `Tom`
+* Multiple names must be seperated by a comma. e.g. `/order find john, hilary`.
+* The order of the keywords does not matter. e.g. `Alice Smith` will match `Smith Alice`
+* Partial match is allowed e.g. `tom` will match `tommy`
+****
 
-// tag::undoredo[]
-=== Undoing previous command : `undo`
+Examples:
 
-Restores the address book to the state before the previous _undoable_ command was executed. +
-Format: `undo`
+* `/deliveryman find john` +
+Returns `john` and `John Doe`
+* `/order find john, alice, tom` +
+Returns `john`, `alice`, `tom`
 
-[NOTE]
-====
-Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit` and `clear`).
-====
+==== Deleting a delivery man : `/deliveryman delete`
+
+Deletes the specified delivery man from the list of delivery men +
+Format: `/deliveryman delete INDEX`
+
+****
+* Deletes a delivery at the specified `INDEX`.
+* The index refers to the index number shown in the displayed delivery men list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
 
 Examples:
 
-* `delete 1` +
-`list` +
-`undo` (reverses the `delete 1` command) +
+* `/deliveryman` +
+`/deliveryman delete 2` +
+Deletes the 2nd delivery man in the list of delivery men.
+* `/deliveryman find tom` +
+`delete 1` +
+Deletes the 1st delivery man in the results of the `find` command.
+
+==== Clearing all delivery men entries : `/deliveryman clear`
+
+Clears all entries from the list of delivery men. +
+Format: `/deliveryman clear`
+
+==== Creating a route : `/route create`
```

Comment:
How will route create know which route to create? You should pass in an order right?